packages/dashboard/src/components/deliveriesTable.tsx

```diff
@@ -345,120 +440,321 @@ export function DeliveriesTable({
     [items, pageItems],
   );
 
-  return (
-    <Stack sx={{ width: "100%" }} spacing={1}>
-      <DataGrid
-        rows={rows}
-        loading={paginationRequest.type === CompletionStatus.InProgress}
-        columns={[
-          {
-            field: "userId",
-            headerName: "User ID",
-            renderCell: ({ row }: GridRenderCellParams<TableItem>) => {
-              const href = `/users/${row.userId}`;
-              return <LinkCell href={href} title={row.userId} />;
+  const renderEmailPreviewBody = (body: string) => {
```

Comment:
It looks like you've copy and pasted much of this code from `packages/dashboard/src/components/messages/emailEditor.tsx` can you share the code instead?